webapp/src/main/webapp/iaf/frankdoc/js/app.js

```diff
@@ -89,12 +90,28 @@ angular.module('iaf.frankdoc').config(['$stateProvider', '$urlRouterProvider', f
 .filter('matchElement', function() {
 	return function(elements, $scope) {
 		if(!elements || elements.length < 1 || !$scope.category) return [];
+		var types = $scope.category.types;
+		var memberNames = [];
```

Comment:
This is duplicate code now because we have a function to do this. I am going to fix this.

Reply:
Done